plugins/modules/template_workflow_manager.py

```diff
@@ -1368,14 +1385,14 @@ def get_template_info(self, template_params):
                     if max_value is not None:
                         _range[j].update({"maxValue": max_value})
                     else:
-                        self.msg = "max_value is mandatory for range under template_params"
+                        self.msg = "max_value is required for range under template_params"
```

Comment:
`The parameter 'max_value' is required for range under 'template_params' but not provided..`